docs/dive-in/2024/02/2024-02-07:configure-ent.md

```diff
@@ -0,0 +1,118 @@
+# WIP NOT FINISHED
+
+# Configure ent and atlas dive-in
+
+Author : Barnabé PILLIAUDIN  
+Helped by: Cédric Gautier
+Date : 2024-02-07
+
+## SUMMARY:
+We want to declare a schema of our database and this schema to be applied to the database.
+But we don't want it to be like dark magic. We want to have some control over it.
+As we are using golang for our backend we think about using gorm and atlas to achieve that.
+
+## SOLUTION:
+
+Atlas offers a clean to do versioned migrations.
```

Comment:
Can you explain?

Reply:
Atlas offers a way to have versionned migrations, which means migrations are in files that are versionned in the repo. It's different that automatic migrations that are run at the start of the api and doesn't generate versionned file.